### DISCUSSION TITLE
$this->resetPage() not working with updated()

### DISCUSSION THREAD

**Discussion by imrancoder** [livewire/livewire#3802] [7. Bugs (archived)] [answered]
`$this->resetPage()` is not working with `updated()` lifecycle method in v2.6 
 
![image](https://user-images.githubusercontent.com/68069779/133894513-b816aabe-a4c4-401a-ba98-503c9ed55ae9.png)

**Comment by ousid** [comment]
What do you want to do exactly?

**Comment by joshhanley** [comment, marked as answer]
@imran-itmajlis thanks for reporting! 🙂

This is a duplicate of #3741. Please see my comment https://github.com/livewire/livewire/discussions/3741#discussioncomment-1352139 for what is going on.

I'm going to mark this one as answered so we can keep discussing the issue in #3741. If you have any ideas can you please post on that discussion.

Hope this helps!

**Comment by joshhanley** [comment]
This has been fixed and tagged in v2.6.5. 🙂

@imran-itmajlis can you please test and let me know if there are any more issues? 

Thanks!

**Reply by imrancoder** [reply]
@joshhanley 
Thanks
It's Resolved in v2.6.5

**Reply by joshhanley** [reply]
@imran-itmajlis great, thanks for testing!
